docs/install/harvester-configuration.md

```diff
@@ -73,6 +73,9 @@ install:
   vip: 10.10.0.19
   vip_hw_addr: 52:54:00:ec:0e:0b
   vip_mode: dhcp
+  cluster_pod_cidr: 172.16.0.0/16
```

Comment:
I'd suggest we use default values here

Reply:
This also reminds me below topic:

The CIDR looks to have three reserved IPv4 ranges for private network: `10.0.0.0/8, 172.16.0.0/12, 192.168.0.0/16`; other CIDR may encounter risks. @ihcsim Is this checked on Harvester-installer? thanks.

https://en.wikipedia.org/wiki/Reserved_IP_addresses

Reply:
@w13915984028 No, we don't have any such checks on the installer. We only make sure they are well-formed CIDRs. It will be nice if these checks can be delegated to RKE2, then RKE2 can tell us exactly what is/isn't acceptable. But AFAICT, [RKE2 also doesn't do this](https://github.com/k3s-io/k3s/blob/646e3135bc9f3afd1f18a5eb13d0a9ffb7f965a1/pkg/cli/server/server.go#L318-L329).

It might not be a bad idea to do this check in the installer, but also feel might not be necessary at this point. WDYT?

Reply:
We may consider to add a suggestion in this document PR, recommend users to set subnets from the above 3 private networks.